sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -669,6 +669,27 @@ def test_check_input_false():
     assert_raises(ValueError, clf.fit, X, y, check_input=False)
 
 
+def test_enet_copy_X_True():
+    X, y, _, _ = build_dataset()
+    X = X.copy(order='F')
+
+    original_X = X.copy()
+    enet = ElasticNet(copy_X=True)
+    enet.fit(X, y, check_input=False)
```

Comment:
I'd use a loop or pytest.mark.parametrize to vary check_input, to clearly show to the reader what invariant you are trying to uphold.

Reply:
Good suggestion. Added pytest.mark.parametrize.